src/library_async.js

```diff
@@ -106,6 +106,7 @@ mergeInto(LibraryManager.library, {
 #endif
 #if ASYNCIFY == 2
       var ASYNCIFY_EXPORTS = {{{ JSON.stringify(ASYNCIFY_EXPORTS) }}};
+      var asyncExports = this.asyncExports = new Set();
```

Comment:
What is `this` in this context?  I don't think we use `this` currently in our JS library code, but maybe we should?

Reply:
`this.` is the same as `Asyncify.`, looks like the old Asyncify=1 code always uses `Asyncify.`. I'll change it.